src/fpm_command_line.f90

```diff
@@ -1459,6 +1536,30 @@ subroutine set_help()
     ' --all             Delete the build without prompting including dependencies.', &
     ' --registry-cache  Delete registry cache.', &
     '' ]
+    help_search=[character(len=80) :: &
+    'NAME', &
+    ' search(1) - search for the package in local and fpm - registry.', &
+    '', &
+    'SYNOPSIS', &
+    ' fpm search', &
+    '', &
+    'DESCRIPTION', &
+    ' Search for packages in the local directory and the fpm-registry, ', &
+    ' supports package search by name, namespace, query (description and README.md)', &
+    ' and license from the registries (local and remote).', &
+    '', &
+    'OPTIONS', &
+    ' --query              Search query (supports wildcard).', &
+    ' --page               Page number for results.', &
+    ' --registry           URL of the registry to query.', &
+    ' --namespace          Namespace of the package', &
+    ' --package            Package name to filter results.', &
+    ' --package_version    Version of the package', &
+    ' --license            License type to filter results.', &
+    ' --limit              Maximum number of results to return.', &
+    ' --sort-by            Field to sort results by (e.g., name).', &
+    ' --sort               Sort order (asc for ascending, desc for descending).', &
```

Comment:
It would be extremely beneficial to add a few examples of fpm commands that can be used to search for packages. That gives the reader a better idea of what the command can do and how it works.

Reply:
I will add some sample queries for each parameter so that it is easy to refer.